app/backend/src/couchers/models.py

```diff
@@ -175,9 +175,10 @@ def display_last_active(self):
 
     @property
     def avatar_url(self):
-        # TODO(aapeli): don't hardcode
-        filename = self.avatar_filename or "default.jpg"
-        return f"{config['MEDIA_SERVER_BASE_URL']}/img/avatar/{filename}"
+        if self.avatar_filename:
+            return f"{config['MEDIA_SERVER_BASE_URL']}/img/avatar/{self.avatar_filename}"
+        else:
+            return ""
```

Comment:
Why not None? Does this mean we are adjusting our business logic to the transport layer's peculiarities?